draft-ietf-quic-transport.md

```diff
@@ -7411,6 +7402,43 @@ The initial contents of this registry are shown in {{iana-error-table}}.
 
 --- back
 
+# Sample Packet Number Encoding Algorithm {#sample-packet-number-encoding}
+
+The pseudo-code in {{alg-encode-pn}} shows how an implementation can select
+an appropriate size for packet number encodings.
+
+The EncodePacketNumber function takes two arguments:
+
+* full_pn is the full packet number of the packet being sent.
+* largest_acked is the largest packet number which has been acknowledged by the
```

Comment:
```suggestion
* largest_acked is the largest packet number that has been acknowledged by the
```

Reply:
Can you also modify the variable definitions in the Decode part to match these?

Reply:
There's already a PR for that, which I was emulating.